cdc/capture/http_handler.go

```diff
@@ -798,7 +742,17 @@ func (h *HTTPHandler) forwardToOwner(c *gin.Context) {
 	defer resp.Body.Close()
 	_, err = bufio.NewReader(resp.Body).WriteTo(c.Writer)
 	if err != nil {
-		c.IndentedJSON(http.StatusInternalServerError, model.NewHTTPError(err))
+		handleError(c, err)
 		return
 	}
 }
+
+func handleError(c *gin.Context, err error) {
+	c.Error(err)
```

Comment:
We should probably print the error's stack trace

Reply:
done